core/src/commandHandler.ts

```
@@ -36,29 +164,41 @@ export async function handleSelfChatCommand(msg: Message) {
-			if (getKey) {
-				if (getKey === 'interests') {
-					const interests = userConfig[getKey];
-					if (interests) {
-						await msg.reply(`Interests: "${interests.join(', ')}"`);
-					} else {
-						await msg.reply(`Key "${getKey}" not found.`);
+			if (parts.length < 2) {
+				await msg.reply('Usage: `!get <key>`');
+				return;
+			}
+			const getKey = parts[1].toLowerCase();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)